docs/docs_settings.py

```diff
@@ -7,26 +7,17 @@
 
 import os
 
-if os.environ['EDX_PLATFORM_SETTINGS'] == 'devstack_docker':
```

Comment:
drive-by clean-up: docker == devstack_docker now, so the import block can be simplified.

Reply:
I don't know anything about this particular change, but I trust you.